pycqed/measurement/openql_experiments/openql_helpers.py

```diff
@@ -13,6 +13,7 @@
 
 output_dir = join(dirname(__file__), 'output')
 ql.set_option('output_dir', output_dir)
```

Comment:
Helpers are not extremely needed anymore, IMO.

Reply:
Same as above.